### ISSUE TITLE
Windows build is broken

### ISSUE THREAD

**Issue by NoUseFreak** [NoUseFreak/letitgo#14]
**Describe the bug**
Currently windows binaries are not build

**Config**
Current project config

**Expected behavior**
Build should have created a windows binary.
Build should have failed if a single binary is not created.

**Debug logs**
```
2 errors occurred:
--> windows/amd64 error: exit status 2
Stderr: # github.com/docker/docker/pkg/system
../../../../pkg/mod/github.com/docker/docker@v1.4.2-0.20190927142053-ada3c14355ce/pkg/system/filesys_windows.go:112:24: cannot use uintptr(unsafe.Pointer(&sd[0])) (type uintptr) as type *"golang.org/x/sys/windows".SECURITY_DESCRIPTOR in assignment
--> windows/386 error: exit status 2
Stderr: # github.com/docker/docker/pkg/system
../../../../pkg/mod/github.com/docker/docker@v1.4.2-0.20190927142053-ada3c14355ce/pkg/system/filesys_windows.go:112:24: cannot use uintptr(unsafe.Pointer(&sd[0])) (type uintptr) as type *"golang.org/x/sys/windows".SECURITY_DESCRIPTOR in assignment
```

**Platform (please complete the following information):**
travisci

**Additional context**
https://travis-ci.org/NoUseFreak/letitgo/builds/603162352

**Comment by NoUseFreak** [comment]
Try implementing github.com/docker/docker directly.

**Comment by NoUseFreak** [comment]
Fixed in 0.8.1
